src/main/java/net/rptools/maptool/model/library/addon/AddOnLibraryManager.java

```diff
@@ -193,4 +210,105 @@ public CompletableFuture<Set<Library>> getLegacyEventTargets(String eventName) {
                 .filter(l -> l.getLegacyEvents().contains(eventName))
                 .collect(Collectors.toSet()));
   }
+
+  /** Initializes the add-on library manager. */
+  public void init() {
+    externalAddOnLibraryManager = new ExternalAddOnLibraryManager(this);
+    String path = AppPreferences.externalAddOnLibrariesPath.get();
+
+    try {
+      externalAddOnLibraryManager.setExternalLibraryPath(Path.of(path));
```

Comment:
@cwisniew This is throwing an NPE when the preference isn't set since the default is `null`.

Reply:
Hmm ok, I will fix it after work tonight (in a couple hours)